server/config/routes.js

```diff
@@ -360,6 +361,13 @@ exports.configure = function (app) {
   app.post('/cashboxes/:id/currencies', cashboxes.currencies.create);
   app.put('/cashboxes/:id/currencies/:currencyId', cashboxes.currencies.update);
 
+  // cash (aux/primary)
+  app.get('/cash', cash.list);
+  app.get('/cash/:uuid', cash.details);
+  app.post('/cash', cash.create);
+  app.put('/cash/:uuid', cash.update);
+  app.delete('/cash/:uuid', cash.debitNote);
```

Comment:
I kept it in as a placeholder for a future "cash_discard" route.  It doesn't actually do anything yet.  If we decide to not use the DELETE request for cash_discard, I'll remove it an make a separate set of routes like `cash/discard/:uuid`.